lifx-core/src/lib.rs

```diff
@@ -1180,6 +1196,15 @@ pub enum Message {
         parameters: [u8; 32],
     },
 
+    /// Message type 510
+    SetExtendedColorZones {
+        duration: u64,
```

Comment:
The [docs](https://lan.developer.lifx.com/docs/changing-a-device#setextendedcolorzones---packet-510) say that this is a `u32` not `u64`

Reply:
Ahhh indeed! I copied from the other nearby structure and that was wrong. I'll fix up from your feedback, thank you!